README.md

```diff
@@ -1,2 +1,27 @@
-# support
-For questions on how to get IPFS up and running smoothly
+# Support
+
+For questions on how to get IPFS up and running smoothly.
```

Comment:
How about mentioning the irc channel, it seems to be to be an active and good resource for help.